src/setup-beam.js

```diff
@@ -175,34 +189,61 @@ async function getRebar3Version(r3Spec) {
 }
 
 async function getOTPVersions(osVersion) {
-  const otpVersionsListing = await get(
-    `https://repo.hex.pm/builds/otp/${osVersion}/builds.txt`,
-  )
-  const otpVersions = new Map()
+  let originListing
+  let pageIdxs
+  if (process.platform === 'linux') {
+    originListing = `https://repo.hex.pm/builds/otp/${osVersion}/builds.txt`
+    pageIdxs = [null]
+  } else if (process.platform === 'win32') {
+    originListing =
+      'https://api.github.com/repos/erlang/otp/releases?per_page=100'
+    pageIdxs = [1, 2, 3]
```

Comment:
Pagination, even if very explicit... I didn't want to deal with looping until I got 404 or `[]`. This seems enough, for the time being.